utils/licenseActions.ts

```diff
@@ -0,0 +1,45 @@
+import { HandlerReturn } from ".";
+import { CtlxClientType } from "./client";
+
+export  const   createLicense = async (client:CtlxClientType, body:any): HandlerReturn =>
+{
+    const license = await client.POST("/v3/licenses", {
+        body,
+      });
+      if (license.error) {
+        throw Error(
+          `License creation failed with error: ${license.error.code} ${license.error.message}. User with ID ${body.userId} has been created.`
+        );
+      }
+      return {
+        message: "License created successfully.",
+        data: { license: license.data },
+        status: 201,
+      };
+}
+
+/** Get ID of a License that has the subscription ID saved in metadata  */
+export async function getLicenseId(client:CtlxClientType,subscriptionId: string, subscriptionIdMetadataKey:string):Promise<string> {
```

Comment:
Do you think returning the whole license here could be better?

Reply:
@azan-n No.
I kind of thought about it initially but I don't think we will need the whole license generally for the operations that we support.